Gemfile.lock

```diff
@@ -709,13 +709,11 @@ DEPENDENCIES
   savon
   sdoc (~> 0.4.0)
   seedbank
-  sentry-raven (= 2.7.4)
+  sentry-raven (= 2.9.0)
   shrine
   shrine-memory
-  sidekiq-ent!
```

Comment:
If you follow the instructions on https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Platform/Vets-API/Sidekiq%20Enterprise%20Setup.md then sidekiq-related gems won't change when you bundle.

Reply:
Whoops.. good lookin out!